debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -972,6 +972,16 @@ public static AutoCreateMode parse(String value, String defaultValue) {
     public static final Field SOURCE_INFO_STRUCT_MAKER = CommonConnectorConfig.SOURCE_INFO_STRUCT_MAKER
             .withDefault(PostgresSourceInfoStructMaker.class.getName());
 
+    public static final Field GROWING_WAL_WARNING_LOG_INTERVAL = Field.create("wal.growing.warning.log.interval")
+            .withDisplayName("Interval between log messages about growing WAL")
+            .withType(Type.INT)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR, 101))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.LOW)
+            .withDescription("Number of received events without sending anything to Kafka which will trigger a WAL backlog growing warning.")
+            .withDefault(10_000)
```

Comment:
Could you please extract the default value into a constant and also add a text into description like `(defaults to ...)`?